easybuild/easyconfigs/g/gompi/gompi-2016.07.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = "Toolchain"
+
+name = 'gompi'
+version = '2016.07'
+
+homepage = '(none)'
+description = """GNU Compiler Collection (GCC) based compiler toolchain,
+including OpenMPI for MPI support."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+gccver = '6.1.0'
+binutilsver = '2.27'
+gccbinver = '%s-%s' % (gccver, binutilsver)
+
+# compiler toolchain dependencies
+dependencies = [
+   ('GCC', gccbinver),  # includes both GCC and binutils
+   ('OpenMPI', '1.10.2', '', ('GCC', gccbinver)),
```

Comment:
@AFCJamie while looking into this in detail, I noticed that OpenMPI 1.10.2 is a *downgrade* compared to the `2016.06` version of `gompi` and `foss` where 1.10.3 is already used, which is rather weird...